js_modules/dagster-ui/packages/ui-core/src/assets/BackfillPreviewModal.tsx

```diff
@@ -50,13 +51,15 @@ export const BackfillPreviewModal = ({
   const totalHeight = rowVirtualizer.getTotalSize();
   const items = rowVirtualizer.getVirtualItems();
 
-  const {data} = useQuery<BackfillPreviewQuery, BackfillPreviewQueryVariables>(
+  const queryResult = useQuery<BackfillPreviewQuery, BackfillPreviewQueryVariables>(
     BACKFILL_PREVIEW_QUERY,
     {
       variables: {partitionNames: keysFiltered, assetKeys},
       skip: !isOpen,
     },
   );
+  useBlockTraceOnQueryResult(queryResult, 'BackfillPreviewQuery');
```

Comment:
I think you mentioned skipping some that are in modals and this one is in a modal?